tests/testthat/test-from_json.R

```diff
@@ -421,15 +421,33 @@ test_that("round trips work", {
 })
 
 test_that("UTF-8 encoding is not mangled", {
-  skip_if_not(Sys.info()['sysname'] == "Windows")
+  skip_if_not(Sys.info()["sysname"] == "Windows")
```

Comment:
I think we can remove this check because we want these tests to succeed on all platforms.